ipynb/Jones99_soil_10_samples.csv

```diff
@@ -0,0 +1,13 @@
+index,sample_name,taxon_id,scientific_name,host_subject_id,description,physical_specimen_location,physical_specimen_remaining,collection_timestamp,latitude,longitude,investigation_type,project_name,experimental_factor,depth,altitude,elevation,geo_loc_name,env_biome,env_feature,env_material,env_package,alt,depth,elev,cur_land_use,cur_vegetation,cur_vegetation_meth,previous_land_use,previous_land_use_meth,crop_rotation,agrochem_addition,tillage,fire,flooding,extreme_event,horizon,horizon_meth,sieving,water_content,water_content_soil_meth,samp_vol_we_dna_ext,pool_dna_extracts,store_cond,link_climate_info,annual_season_temp,annual_season_precpt,link_class_info,fao_class,local_class,local_class_meth,soil_type,soil_type_meth,slope_gradient,slope_aspect,profile_position,drainage_class,texture,texture_meth,ph,ph_meth,tot_org_carb,tot_org_c_meth,tot_nitro,tot_n_meth,microbial_biomass,microbial_biomass_meth,link_addit_analys,extreme_salinity,salinity_meth,heavy_metals,heavy_metals_meth,al_sat,al_sat_meth,misc_param
```

Comment:
Yes. The "index" column was purposely left in because it indicates whether the row is "Requirement" information, "Format" information, or a blank row for sample information.